agent-install/agent-install.sh

```diff
@@ -1823,7 +1842,7 @@ function loadClusterAgentImage() {
             if [[ -z $image_tag ]]; then
                 image_tag='latest'
             fi
-            local image_path="openhorizon/amd64_anax_k8s:$image_tag"
+            local image_path="openhorizon/${ARCH}_anax_k8s:$image_tag"
```

Comment:
- On a rhel edge cluster ARCH will be `x86_64` instead of `amd64`, right? I don't think we have published `openhorizon/x86_64_anax_k8s:<tag>`
- Did you test this on a ppc64le edge cluster?

Reply:
Yes, that was tested on Power platform. For all images I've used private registry, with reassigning paths/tags.
@bmpotter Would you recommend to convert image names in proper way for multi-arch usage to support, or just apply if/else based upon only supported distro/archs here ? I can then fix that.

Reply:
The `ARCH` variable is set to what is needed for the packages (debs or RPMs) on the current host. I think we need a different variable for `IMAGE_ARCH`, because there is no reason we should have to build both an `amd64` and `x86_64` version of the anax and anax_k8s images. `IMAGE_ARCH` should be set to `amd64` on an amd64 or x86_64 host (regardless of when debian or RHEL) and set to `ppc64le` on Power.

Reply:
Done, pls recheck. P.S. No global variable was setup there since it shouldn't be reused or redeclared.